ocs_ci/ocs/cosbench.py

```diff
@@ -637,3 +637,36 @@ def get_performance_result(self, workload_name, workload_id, size):
         logger.info(
             f"Bandwidth data present in {log_path}/{workload_name}-{size}-bandwidth.csv"
         )
+
+    def cosbench_full(self):
+        """
+        Run full Cosbench workload
+        """
+        bucket_prefix = "bucket-"
+        buckets = 10
+        objects = 50
+
+        # Operations to perform and its ratio(%)
+        operations = {"read": 50, "write": 50}
+
+        # Deployment of cosbench
+        self.setup_cosbench()
+
+        # Create initial containers and objects
+        self.run_init_workload(
+            prefix=bucket_prefix, containers=buckets, objects=objects, validate=True
+        )
+
+        # Run main workload
+        self.run_main_workload(
+            operation_type=operations,
+            prefix=bucket_prefix,
+            containers=buckets,
+            objects=objects,
+            validate=True,
+        )
+
+        # Dispose containers and objects
+        self.run_cleanup_workload(
+            prefix=bucket_prefix, containers=buckets, objects=objects, validate=True
```

Comment:
Do add try ..... except to capture the exception in-case of failures.

Reply:
Discussed offline with Ram, I am handling the failures case in the fixture teardown. No changes are required here.